examples/09_Kotlin_JS/06_HtmlBuilder.md

```diff
@@ -1,10 +1,8 @@
 # Html Builder
 
-This is an example of a Type-Safe Groovy-style Builder
-
-Builders are good for declaratively describing data in your code.
-In this example we show how to describe an HTML page in Kotlin
+Kotlin provides you with a option to describe structured data in a declarative style with _builders_.
 
+Below is an example of a type-safe Groovy-style builder. In this example we show how to describe an HTML page in Kotlin.
```

Comment:
Below is an example of a type-safe Groovy-style builder. In this example, we will describe an HTML page in Kotlin.